### PR TITLE
adds spent subtypes for some ammo and adds them to the waystation and the faceoff space ruins.

### PR DESCRIPTION
**Pull request by rageguy505**
## About The Pull Request
Adds spent subtypes for .357, .45 acp and buckshot.
Replaces live ammo with the spent subtype on the waystation and the faceoff space ruins.
## Why It's Good For The Game
I talked with a map maker and he didn't intend for 60+ live .357 rounds on his map.
I assumed its the same for the waystation.
## Changelog
:cl:

fix: Replaced live ammo found that was unintended on the waystation and the faceoff with spent subtypes.
/:cl:

### CODE REVIEW

**Comment by SyncIt21** [comment]
Again be more specific about what map files you are editing. 

> adds them to some ruins.

From the files changed you are editing the_faceoff.dmm, waystation.dmm & north_star.dmm. Please put these map names in your PR title and description.

**Comment by rageguy505** [comment]
> 

this doesn't touch the northstar, but ill update the rest

File: _maps/map_files/NorthStar/north_star.dmm [hunk 1]
```diff
@@ -16637,6 +16637,16 @@
 	},
 /turf/open/floor/iron/smooth_large,
 /area/station/hallway/secondary/entry)
+"emV" = (
+/obj/structure/closet/secure_closet/brig/genpop,
```
Comment (SyncIt21) [inline _maps/map_files/NorthStar/north_star.dmm:16641]: Why are you adding genpop closets? wait did you create a new branch from your previous PR #74924?

Comment (SyncIt21) [reply _maps/map_files/NorthStar/north_star.dmm:16641]: Now looking back at that PR the changes are exactly the same

Comment (rageguy505) [reply _maps/map_files/NorthStar/north_star.dmm:16641]: I don't know why, but my GitHub shows all the changes I've done when I open a new pr.

Comment (SyncIt21) [reply _maps/map_files/NorthStar/north_star.dmm:16641]: You need to create a new branch for every PR, you have reused the same branch from your last PR hence all changes made back then are showing up here. For this one the solution is pretty simple, just copy & paste the original northstar.dmm from a new branch to undo all changes to it.

**Comment by SyncIt21** [comment]
Yup that fixed the problem

**Comment by Rex9001** [comment]
Will holdoff on my faceoff changes until this is merged

**Pull request merged**